src/components/shared/SearchBar/index.test.tsx

```diff
@@ -13,18 +14,193 @@ describe('The Search Bar component', () => {
     expect(component.find(`input[name="${fixture}"]`).length).toEqual(1);
   });
 
-  it('triggers on onChange action', () => {
-    const fixture = jest.fn();
-    const event = {
-      target: {
-        value: 'EASi'
-      }
-    };
-
+  it('prevents the default action of submitting a form', () => {
+    const spy = jest.fn();
     const component = mount(
-      <SearchBar name="test-name-attr" onSearch={fixture} />
+      <SearchBar name="test-name-attr" onSearch={() => {}} />
     );
-    component.find('.usa-input').simulate('change', event);
-    expect(fixture).toHaveBeenCalled();
+    const searchBtn = component.find('[data-testid="search-bar-search-btn"]');
+    component.find('.usa-input').simulate('change', { target: { value: '' } });
+    searchBtn.simulate('submit', {
+      preventDefault: () => {
+        spy();
+      }
+    });
+    expect(spy).toHaveBeenCalled();
+  });
+
+  describe('The default Search Bar w/o autocomplete', () => {
```

Comment:
The search bar was originally developed without the autocomplete functionality. Because the autocomplete functionality depends on certain props, I decided to maintain the ability to have a search bar without autocomplete in case those props are not passed.

The search bar is a reusable component and I imagine it could be used in a different part of the application without the autocomplete functionality.